person.rb

```diff
@@ -21,6 +22,11 @@ def correct_name
     @name
   end
 
+  def add_rentals()
+    rental = Rental.new(date, self, book)
+    @rentals.push(rental)
+  end
+
```

Comment:
- [ ] You need to provide the `book` and `date` as parameters in this method

- [ ] Let us be mindful of the arguments order.

  ```
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
  ```

- [ ] Kindly remove the code on line 27 as it is making a duplicate of an already added rental.